seq2science/rules/deseq2.smk

```diff
@@ -47,7 +46,7 @@ rule deseq2:
     log:
         expand("{log_dir}/deseq2/{{assembly}}-{{contrast}}.diffexp.log", **config),
     message:
-        explain_rule("deseq2")
+        EXPLAIN.get("deseq2", "")
```

Comment:
wouldn't it have been easier to make this a defaultdict?

Reply:
it is now :)